src/workerd/io/actor.h

```diff
@@ -0,0 +1,118 @@
+#include "worker.h"
+#include "worker-interface.h"
+#include "io-gate.h"
+#include "actor-cache.h"
+#include <workerd/api/global-scope.h>
+
+namespace workerd {
+
+// TODO(now): Rename to Worker::Actor
```

Comment:
Nit: We use `TODO(now)` for things that should be done before this PR lands. Perhaps `TODO(cleanup)` or `TODO(soon)`?

Reply:
Sorry, converted the PR to a draft when I noticed that was an option. There's still quite a bit of work to do here, I just wanted to offer the DO team a cleaner way of making suggestions before we get too far into implementing this approach.